### DISCUSSION TITLE
GitHub Code Search does not return result

### DISCUSSION THREAD

**Discussion by hellozwinger** [community/community#72266] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hi,

When I use "code search" to search with some keywords, it does not return the expected result.
Eg:
The file "https://github.com/hellozwinger/helloworld/blob/master/File_A" contains following snippet:
              tom:123456@ccccc.com/should_be_hitted
Then when I search with "should_be_hitted", nothing returns.

![image](https://github.com/community/community/assets/22537401/5427936f-a868-4491-b4ca-d8015bb0a424)

I found someone reported such issue in 2022, and seems the "code search" function works after some fix.

But I met with the same issue now, would someone help to take a look?

Thanks!

**Comment by look** [comment, marked as answer]
Not all repositories are indexed. To trigger indexing, you can search within a repository specifically and then wait a few minutes. I did this in your repository and it is now indexed and returns a result for your query.

**Reply by Gargo** [reply]
@look could you please check it for this repo?
https://github.com/search?q=repo%3AShamKumarS/EditableTableViewDemo%20reuseIdentifier&type=code
nothing found but "reuseIdentifier" exists for example here:
https://github.com/ShamKumarS/EditableTableViewDemo/blob/main/EditableTableViewDemo/Views/Task%20Listing/TaskListingViewController.swift

**Reply by look** [reply]
It's working for me, please try again. As I mentioned, there is a delay after indexing before the code is searchable.

**Comment by Mohammad222PR** [comment]
Hi 
Check Query Syntax: Ensure that the syntax of your search query is correct. GitHub's code search supports various filters and qualifiers, so make sure you are using them appropriately.

Try Different Keywords: Experiment with different keywords or variations of your search query to see if you get the expected results. Sometimes, changing the search terms can help identify if the issue is specific to certain keywords.

Browser Cache: Clear your browser cache and try the search again. Cached data might sometimes interfere with the search results.

Report the Issue: If the problem persists, it's a good idea to report the issue to GitHub support. Provide details about your search query, the expected result, and what you are actually getting. Mention that you've encountered a similar issue reported in 2022, and despite fixes, you're still facing the problem.

To report the issue, you can go to GitHub's support or help section and follow the steps for submitting a bug report or reaching out to their support team.

**Comment by yueyuecompany23** [comment]
I do have problem about bulk queries.
